server/uno.go

```diff
@@ -245,6 +245,17 @@ func dealCards(game *model.Game) (*model.Game, error) {
 	// draw a card for the discard
 	var drawnCard model.Card
 	game, drawnCard = drawTopCard(game)
+	
+	// ensure that this first card is a number card 
+	for !isNumberCard(drawnCard) {
```

Comment:
I think this would be a lot cleaner if this was done with a regular `while` loop...

Reply:
Golang does not have a while loop

Reply:
Sorry, missed that this was on the server side...